### PR TITLE
Add reference to awake to russian/light-fire/so-scary/so-scary.md

### PR DESCRIPTION
**Pull request by valery-trigubov**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1832)

### CODE REVIEW

File: russian/light-fire/so-scary/so-scary.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Вы остаетесь в комнате. Расплавленный зефир потихоньку затапливает помещение.
 И вот горячая сладкая жидкость уже достигла ваших колен. Но вы не отчаиваетесь
-и успеваете съедать зефир по мере его поступления.
\ No newline at end of file
+и успеваете съедать зефир по мере его поступления.
+
+[Вы просыпаетесь.](awaken/awaken.md)
```
Comment (casey-collab) [inline russian/light-fire/so-scary/so-scary.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by valery-trigubov** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**